package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "citizen-app",
+  "version": "1.0.0",
+  "description": "This app is for the reporting of non-emergency civic issues to a centralized database.",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/malakivandas/citizen-app.git"
+  },
+  "author": "Malaki Vandas",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/malakivandas/citizen-app/issues"
+  },
+  "homepage": "https://github.com/malakivandas/citizen-app#readme",
+  "scripts": {
+    "postinstall": "cd client && npm install && cd ../server && npm install",
```

Comment:
I would also `cd ..` to go back up to the root directory (I imagine when you run this in the root, your shell will end up above, but maybe this doesn't happen).

Reply:
Nope doesn't happen for me, but I'll add it in just in case.

Reply:
This didn't end up causing an issue right?